### PR TITLE
remove wlp entries from jsp bundles

### PR DESCRIPTION
**Pull request by chungngoops**
#26903

### CODE REVIEW

**Comment by volosied** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.webcontainer.servlet.3.1_fat,com.ibm.ws.webcontainer.servlet.4.0_fat

File: dev/io.openliberty.webcontainer.servlet.6.0.internal/bnd.bnd [hunk 1]
```diff
@@ -1,23 +1,18 @@
 #*******************************************************************************
-# Copyright (c) 2022 IBM Corporation and others.
+# Copyright (c) 2023 IBM Corporation and others.
```
Comment (volosied) [inline dev/io.openliberty.webcontainer.servlet.6.0.internal/bnd.bnd]:
```suggestion
# Copyright (c) 2022, 2023 IBM Corporation and others.
```

File: dev/io.openliberty.webcontainer.servlet.6.0.internal.factories/bnd.bnd [hunk 2]
```diff
@@ -1,14 +1,11 @@
 #*******************************************************************************
-# Copyright (c) 2022 IBM Corporation and others.
+# Copyright (c) 2023 IBM Corporation and others.
```
Comment (volosied) [inline dev/io.openliberty.webcontainer.servlet.6.0.internal.factories/bnd.bnd]:
```suggestion
# Copyright (c) 2022, 2023 IBM Corporation and others.
```

File: dev/io.openliberty.session.6.0.internal/bnd.bnd [hunk 3]
```diff
@@ -1,23 +1,18 @@
 #*******************************************************************************
-# Copyright (c) 2022 IBM Corporation and others.
+# Copyright (c) 2023 IBM Corporation and others.
```
Comment (volosied) [inline dev/io.openliberty.session.6.0.internal/bnd.bnd]:
```suggestion
# Copyright (c) 2022, 2023 IBM Corporation and others.
```

File: dev/com.ibm.ws.jsf.jsp.shared/bnd.bnd [hunk 4]
```diff
@@ -1,24 +1,18 @@
 #*******************************************************************************
-# Copyright (c) 2022 IBM Corporation and others.
+# Copyright (c) 2023 IBM Corporation and others.
```
Comment (volosied) [inline dev/com.ibm.ws.jsf.jsp.shared/bnd.bnd]:
```suggestion
# Copyright (c) 2022, 2023 IBM Corporation and others.
```

**Comment by volosied** [comment]
Add each suggestion to the batch (I'm just keeping the original year). 

I will look at how to run a build since Libby denied my request.

**Comment by chungngoops** [comment]
> Add each suggestion to the batch (I'm just keeping the original year).
> 
> I will look at how to run a build since Libby denied my request.

Ah okey, I will update my PR.
I sent my CLA, not sure when it will be accepted.

**Comment by volosied** [comment]
Thanks for the update. I'm running a build on my repository using your commits: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_mDl6sIPkEe6tffXfLjKGSA&tab=com.ibm.team.build.web.ui.internal.editors.result.properties.PropertiesProvider

Once it passes, we can merge (assuming all other checks pass)

Great work! Thanks for your help here :)

**Comment by volosied** [comment]
#libby

**Comment by volosied** [comment]
Successful build: 

image

**Review by volosied — APPROVED** [review]

**Review by pnicolucci — APPROVED** [review]

**Comment by volosied** [comment]
#libby

**Pull request merged**